libmozevent/utils.py

```diff
@@ -175,16 +184,31 @@ class AsyncRedis(object):
     @staticmethod
     async def connect():
         if AsyncRedis.redis.get(None) is None:
+            retry_on_error = [
+                ConnectionError,
+                ExecAbortError,
+                PubSubError,
+                RedisError,
+                ResponseError,
+                TimeoutError,
+                WatchError,
+            ]
+
+            retries = 5
```

Comment:
Make this uppercase since it is a constant, e.g. call it `MAX_RETRIES`.

Also, let's use 7 instead of 5 to allow more retries.